lektor/admin/static/js/views/EditPage.jsx

```diff
@@ -130,6 +130,29 @@ class EditPage extends RecordEditComponent {
   }
 
   saveChanges () {
+    const path = this.getRecordPath()
+    const alt = this.getRecordAlt()
+    const newData = this.getValues()
+    utils.apiRequest('/rawrecord', { json: {
+      data: newData, path: path, alt: alt },
+    // eslint-disable-next-line indent
+      method: 'PUT' }, makeRichPromise)
+      .then((resp) => {
+        this.setState({
+          hasPendingChanges: false
+        }, () => {
+          this.transitionToAdminPage('.edit', {
+            path: this.getUrlRecordPathWithAlt(path)
+          })
+          window.location.reload()
+          if (!resp.okay) {
```

Comment:
Why is this statement inverted?

Reply:
I found that in a similar spot - but I've since removed it to handle things more in line with elsewhere in the code.